app/javascript/controllers/sidebar_group_controller.js

```diff
@@ -16,4 +22,63 @@ export default class extends Controller {
       }
     })
   }
+
+  // For group list with anchor links where toggleShow() does not work because
+  // active class does not get triggered in sidebar_helper.rb
+  toggleShowAnchorList () {
+    this.titleTarget.classList.remove('collapsed')
```

Comment:
The `active_class` is in the `SidebarHelper.active_class` module. Let's adapt the logic there to make it work with the anchors.